Find a functionally equivalent code:android.view.inputmethod.InputMethodManager.clearBindingLocked:COMMENT
Method Modifier: 
Comment:/**
 * Reset all of the state associated with being bound to an input method.
 */

@@ -1,4 +1,6 @@
 {
+    if (DEBUG)
+        Log.v(TAG, "Clearing binding!");
     clearConnectionLocked();
     setInputChannelLocked(null);
     mBindSequence = -1;

